src/grub.cfg

```diff
@@ -44,8 +44,7 @@ fi
 # which is used in the kernel command line.
 set ignition_firstboot=""
 if [ -f "/ignition.firstboot" ]; then
-    # default to dhcp networking parameters to be used with ignition
-    set ignition_network_kcmdline='rd.neednet=1 ip=dhcp,dhcp6'
```

Comment:
Who will inject this default later on, and how will overriding work?

Reply:
Answering to myself: the whole plan is at https://github.com/coreos/fedora-coreos-tracker/issues/460.